test/v2/services/RetrieveForeignPropertyAdjustmentsServiceSpec.scala

```diff
@@ -33,7 +33,7 @@ import scala.concurrent.Future
 class RetrieveForeignPropertyAdjustmentsServiceSpec extends UnitSpec {
 
   private val nino = Nino("AA123456A")
-  private val correlationId = "X-123"
+  private implicit val correlationId = "X-123"
```

Comment:
Please remove once you have added the **implicit val correlationId: String = "X-123"** to the ServiceSpec and this class extends it instead (Change from UnitSpec to ServiceSpec)